Sources/Talkshoplive/Utils/Helpers/Date_Extension.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 extension Date {
     var milliseconds: TimeInterval {
-        return self.timeIntervalSince1970 * 1000.0
+        return self.timeIntervalSince1970 * 10000000
```

Comment:
Milliseconds should be x 1000. This looks more of nano seconds?

Reply:
@daman-talkshoplive  If I pass with * 1000 , it's not working , it's giving me blank array.

Reply:
@daman-talkshoplive yes it's takes nanoseconds. Fixed for function naming conversation.